packages/client/screens/ContactsSettingsScreen.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function ContactsSettingsScreen() {
```

Comment:
A better name, shouldn't be tied to the settings page

```suggestion
export default function ContactsListScreen() {
```